La-Mula-App-main/src/main/kotlin/com/lamulaapp/domain/Login.kt

```diff
@@ -0,0 +1,36 @@
+package com.lamulaapp.domain
+
+import jakarta.persistence.*
+import java.util.*
+
+@Entity
+@Table (name = "login")
+data class Login(
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    val id: UUID? = null,
+
+    @OneToOne(cascade = [CascadeType.ALL], fetch = FetchType.LAZY)
+    @JoinColumn(name = "id_user", referencedColumnName = "id_user")
+    val iduser: Users? = null,
+
+    @Column( name = "username")
+    val username: String,
+
+    @Column( name = "password")
+    val password: String,
+
+    @Column( name = "create_at")
+    @Temporal( TemporalType.DATE)
+    var createdate: Date,
+
+    @Column( name = "create_by")
+    var createby: String,

```

Comment:
Recuerda que en Kotlin/Java por estandar las variables se nombran en camelCase. 
realiza ese cambio en todo lado donde exista mas de una palabra en todas las variables.

(createdAt, createdBy, updateAt, updateBy) entre otras.

```suggestion
    var createBy: String,

```